build.sbt

```diff
@@ -16,7 +16,8 @@ libraryDependencies ++= Seq(
   "org.yaml" % "snakeyaml" % "1.11",
   "com.typesafe.play" %% "anorm" % "2.4.0",
   "com.zaxxer" % "HikariCP" % "2.4.5",
-  "com.indeed" % "util-mmap" % "1.0.20"
+  "com.indeed" % "util-mmap" % "1.0.20",
+  "com.h2database" % "h2" % "1.4.196" /*% "test"*/
```

Comment:
You probably can uncomment `% "test"` now because you're only using H2 for testing. This will make `assembly` command skip h2 dependency so it won't be part of the distribution of tempest.